active/fellowship.md

```diff
@@ -2,30 +2,67 @@
 
 ## Scope of responsibilities
 
-The Fellowship Working Group manages the operation of the [Django Fellowship Program](https://www.djangoproject.com/fundraising/#fellowship-program). They select candidates to fill the fellowship, and act as line managers during the operation of the program.
+The Fellowship Working Group manages the [Django Fellowship Program](https://www.djangoproject.com/fundraising/#fellowship-program). This includes:
+
+- Interviewing candidates for open Fellowship slots and making hiring recommendations to the Board. This includes developing and maintaining interview and selection scripts, exercises, and rubrics.
+- Along with the Fellows themselves, maintaining the job description, outline of duties, and any other similar process documentation for the Fellow role (a "job manual", if you will). By and large these don't currently (as of Spring 2024), so the first group operating under this new charter will be responsible for starting to create this documentation. The Fellows and WG can maintain and update these without explicit Board approval, but should report major changes to the Board for review.
+- Designating one member of the committee to act as the line manager for active fellows. See [Fellow Management](#fellow-management), below, for details of this role.
+- Acting as an HR committee for Fellows. In the event of a complaint about a Fellow, or if there are performance issues, the Fellowship-WG will investigate, in concert with the [Code of Conduct WG](code-of-conduct.md) if necessary, and present findings and a recommendation to the board for action.
+  - As with the CoC-WG, the Fellowing-WG has the power to unilaterally suspend Fellows, with pay, pending an investigation. This power should only be used in egregious circumstances, where waiting on investigation and/or board action might plausibly be harmful.
+- Acting as an escalation point if a Fellow has a disagreement with their designated manager (or that person isn't available, etc). The escalation path for Fellows is: designated manager -> full Fellowship WG -> Board.
```

Comment:
Escalating from designated manager to full Fellowship WG is a bit odd as the manager is included in the Fellowship WG
This would work is the designated manager is not available but if there is an issue with them, is an awkward escalation approach

Reply:
What would you suggest -- going straight to the Board, perhaps? Or clarify that they _can_ escalate to the WG, but can also choose to skip them and go right to the Board?

My intent here is to make sure that Fellows don't get "stuck" with a manager relationship that's not working for them, and that they have ways to get help if something goes wrong.

Reply:
I would suggest that it's just to one of the other WG members (since we'll always have at least two) rather than the full set.

Reply:
> My intent here is to make sure that Fellows don't get "stuck" with a manager relationship that's not working for them, and that they have ways to get help if something goes wrong.

I really appreciate having this, thank you

> I would suggest that it's just to one of the other WG members (since we'll always have at least two) rather than the full set.

This makes sense to me 👍

Reply:
WFM. I also re-worded this to say that Fellows _can_ escalate to (various options), so as not to imply they must follow a particular chain. Options, not requirements.